Wikipedia/Code/MWKImage.h

```diff
@@ -46,6 +46,22 @@
 
 
 /**
+ * Width of the original image file. For example, if this image is a thumbnail, this is the width of the original image.
+ */
+@property (copy, nonatomic) NSNumber* originalFileWidth;
```

Comment:
Are we handling the case of where previously serialized MWKImages won't have this information? Is there anything to do? Or any side affects to that?